cloudman/values.yaml

```diff
@@ -62,6 +62,89 @@ helmsman_config:
+      template: |
+        ingress:
+          enabled: true
+          access_path: "/{{context.project.access_path}}/nextcloud"
+          hosts:
+            - name: {{context.domain}}
+              path: /{{context.project.access_path}}/nextcloud
+
+        nextcloud:
+          configs:
+            path.config.php: |
+              <?php
+              $CONFIG = array (
+                'overwriteprotocol' => 'https',
+                'overwritewebroot' => '{{context.project.access_path}}/nextcloud',
+                'overwritehost'     => '{{context.domain}}',
+                'overwrite.cli.url' => 'https://{{context.domain}}/{{context.project.access_path}}/nextcloud',
+                'skeletondirectory' => '',
+                'allow_user_to_change_display_name' => false,
+                'lost_password_link' => 'disabled',
+                'oidc_login_default_quota' => 1000000000,
+                'oidc_login_client_id' => '{{context.oidc_client_id}}',
+                'oidc_login_client_secret' => '{{context.oidc_client_secret}}',
+                'oidc_login_provider_url' => 'https://{{context.domain}}/auth/realms/master',
+                'oidc_login_logout_url' => 'https://{{context.domain}}/auth/realms/master/protocol/openid-connect/logout?redirect_uri=https%3A%2F%2F{{context.domain}}%2F{{context.project.access_path}}%2Fnextcloud%2F',
+                'oidc_login_scope' => 'openid profile',
+                'oidc_login_auto_redirect' => true,
+                'oidc_login_redir_fallback' => true,
+                'oidc_login_use_external_storage' => true,
+                'oidc_login_disable_registration' => false,
+                'oidc_login_attributes' => array(
+                  'id' => 'preferred_username',
+                  'mail' => 'email',
+                ),
+              );
+
+          host: "{{context.domain}}"
+          extraVolumes:
+            - name: project-data
+              persistentVolumeClaim:
+                claimName: "{{context.project.namespace}}-project-data-pvc"
+            - name: public-data
+              persistentVolumeClaim:
+                claimName: "{{context.project.namespace}}-public-data-pvc"
+
+
+          extraVolumeMounts:
+            - name: project-data
+              mountPath: /gvl/projects/current
+            - name: public-data
+              mountPath: /gvl/public
+
+        persistence:
+          enabled: false
+
+        lifecycle:
+          postStartCommand:
+            - su
+            - www-data
+            - -s
+            - /bin/bash
+            - -c
+            - > 
+              PATH=/var/www/html:$PATH;
+              occ app:update --all;
+              occ upgrade;
+              occ app:enable files_external;
+              occ app:disable photos;
+              occ app:install oidc_login;
+              occ app:install external;
+              occ files_external:create -c "datadir=/gvl" gvl local null::null;
```

Comment:
I think we'll need to revisit mounting `/gvl`. This is a container path right? The moment the container is restarted, this data will disappear, yet, it'll be the default folder that users will upload to naturally. Perhaps with our new plans, this root folder should actually be the "project" folder, and all other apps will simply mount webdav anyway. ping @afgane @almahmoud

Reply:
I just found out there's a config option called ` 'filesystem_check_changes' => 1`. Maybe we can mount `/gvl` as the user directory instead?